docs/source/DeveloperDocumentation/TaskEngineAPI.md

```diff
@@ -466,6 +466,49 @@ Successful Response:
 
 </details><br /><br />
 
+### POST: `/push_jobs`
+
+This endpoint is used to force queued jobs to run. For example if a setting such as the maximun number of cuncurrent jobs is updated without using the api while multiple jobs are running the Task engine will keep running the previous amount of maximun cuncurrent jobs and so push jobs needs to be called so the updated stting is used. Successful requests will return an `200 ok` status. An error message is returned when a the request fails submission fails.
+
+<details>
+
+<summary>Details</summary>
+<br />
+
+**Requires Authentication: Yes**<br />
+**Requires Administrator Privileges: Yes**<br />
```

Comment:
If this requires Admin privileges, can you confirm it is an endpoint that clients can reach and execute? If not, should these docs be moved to the Internal API endpoints in Confluence?

Reply:
it can be accessed by clients it is just that admins can pass a name query in it to push_jobs for specific clients

Reply:
@Rodrigo010497 perhaps you can add a clarifying statement to that effect rather than "Require: Yes"?

Reply:
I would remove this statement. Standalone clients can access it and that's what we care about in the public docs. If an admin user (user on Vualto's VPN) can do something extra, it should be documented in confluence.